Base/QTApp/qSlicerApplicationMainWrapper.cxx

```diff
@@ -24,11 +24,16 @@
 #if defined (_WIN32) && !defined (Slicer_BUILD_WIN32_CONSOLE)
 # include <windows.h>
 # include <vtksys/Encoding.hxx>
+# include<shobjidl_core.h>
```

Comment:
AppUserModelID should be set both in case of console and non-console application (not just when `!defined (Slicer_BUILD_WIN32_CONSOLE)`).

Reply:
The solution I proposed for this one is not so elegant. Please advise.